coap-gateway/swagger.yaml

```diff
@@ -0,0 +1,770 @@
+                  pri: 2
+                - ep: coaps://[fe80::b1d6]:1122
+                - ep: coaps+tcp://[2001:db8:a::123]:2222
+                  pri: 3
+            - href: /temperature
+              rt:
+                - oic.r.temperature
+              if:
+                - oic.if.s
+                - oic.if.baseline
+              p:
+                bm: 3
+              eps:
+                - ep: coaps://[[2001:db8:a::123]:2222
+          schema:
+            $ref: '#/definitions/slinklist'
+  /api/v1/devices/{deviceId}/{href}:
+    get:
+      parameters:
+        - name: deviceId
+          in: path
+          required: true
+          type: string
+          description: Device ID
+        - name: href
+          in: path
+          required: true
+          type: string
+          description: Resource href
+        - name: observe
+          in: header
+          type: integer
+          description: A value of 0 indicates registering an observation, while 1 signifies deregistering an observation. As the resource undergoes changes, the response observation will be transmitted as a sequence in the observe header of the responses.
+        - $ref: '#/parameters/interface'
+      tags:
+        - Device Resource
+      summary: Get or observer device resource from IoT Hub.
+      produces:
+        - application/vnd.ocf+cbor
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+    post:
+      parameters:
+        - name: deviceId
+          in: path
+          required: true
+          type: string
+          description: Device ID
+        - name: href
+          in: path
+          required: true
+          type: string
+          description: Resource href
+        - $ref: '#/parameters/interface'
+      tags:
+        - Device Resource
+      summary: Update/Create device resource through IoT Hub.
+      produces:
+        - application/vnd.ocf+cbor
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+    delete:
+      parameters:
+        - name: deviceId
+          in: path
+          required: true
+          type: string
+          description: Device ID
+        - name: href
+          in: path
+          required: true
+          type: string
+          description: Resource href
+      tags:
+        - Device Resource
+      summary: Delete device resource through IoT Hub.
+      produces:
+        - application/vnd.ocf+cbor
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+  /x.plgd.dev/time:
+    get:
+      tags:
+        - Time
+      summary: Get the current time
+      produces:
+        - application/vnd.ocf+cbor
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/TimeResponse"
+parameters:
+  interface:
+    in: query
+    name: if
+    type: string
+    enum:
+      - oic.if.baseline
```

Comment:
only baseline is supported by all that use the param?

ie:
/oic/sec/account
/oic/sec/session
/oic/sec/tokenrefresh
/oic/rd
/api/v1/devices/{deviceId}/{href}

Reply:
Fixed